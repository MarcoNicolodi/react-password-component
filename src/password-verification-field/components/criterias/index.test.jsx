import React from "react";
import renderer from "react-test-renderer";
import Criterias from "./";

describe("Criterias component", () => {
  it("should match snapshot", () => {
    const rules = [
      {
        rule: "Pelo menos 6 caracteres",
        fulfilled: false
      },
      {
        rule: "Pelo menos 1 letra",
        fulfilled: true
      },
      {
        rule: "Pelo menos 1 n√∫mero",
        fulfilled: true
      }
    ];
    const tree = renderer.create(<Criterias criterias={rules} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
