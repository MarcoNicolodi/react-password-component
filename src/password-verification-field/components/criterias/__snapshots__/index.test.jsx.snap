// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Criterias component should match snapshot 1`] = `
<div
  className="strengh-criteria"
>
  <div
    className="criteria"
  >
    <div
      className="round unfulfilled"
    />
    <p>
       
      Pelo menos 6 caracteres
       
    </p>
  </div>
  <div
    className="criteria"
  >
    <div
      className="round fulfilled"
    />
    <p>
       
      Pelo menos 1 letra
       
    </p>
  </div>
  <div
    className="criteria"
  >
    <div
      className="round fulfilled"
    />
    <p>
       
      Pelo menos 1 n√∫mero
       
    </p>
  </div>
</div>
`;
